keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+
+When a release cycle stage such as enhancement freeze or code freeze occurs, the issue is updated to reflect inclusion or removal from the milestone, and we again leverage comments and manually applied labels to convey status; as an example, consider [Build-in API Types Misses Code Freeze](https://github.com/kubernetes/enhancements/issues/1929#event-3991979549).
+
+Additionally, when an exception is required, this is [handled via email](https://groups.google.com/g/kubernetes-sig-release/c/eCk4Xyb58GQ/m/tjgEstjyAQAJ) and linked back to the issue via additional comments, such as [Built-in API Types Defaults Exception Granted](https://github.com/kubernetes/enhancements/issues/1929#issuecomment-727043537).
+
+Over time, this process becomes harder and harder to navigate, leading to decreased process clarity, frustration by contributors, and increased toil for the release team.
+
+To streamline this process and increase transparency, we will add a new Git-based artifact, hereafter referred to as a receipt. This receipt represents an opt-in promise/soft commitment from a SIG about KEPs they're submitting to a specific release and will allow all tracking to occur via Git commits rather than via issue comments, PR references, and other non-auditable mechanisms.
+
+### Goals
+
+* Support better KEP validation and automation through Git-based process
+* Reduce friction and toil early in a release cycle
+* Eliminate instances where Enhancements aren't tracked due to miscommunication
+* Encourage SIGs to update KEPs proactively and plan for release cycles
+* Improve the Exception process turning exception requests into a PR by the SIG with proper approvals
+* Increase KEP / Release transparency
+* Encourage better KEP implementation history updates
+
+
+### Non-Goals
+
+* Generation of receipts for release cycles already closed (Eg. v1.20 and below)
+  * Other than tooling development purposes
+* Generating receipts by RT Enhancements Team on behalf of SIGs and KEP Authors
+
+
+## Proposal
+
+The opting-in process to a release will accept a specified file format (hereafter referred to as a `receipt`) containing metadata about the KEP being enrolled. There would be one receipt per KEP per release cycle.
+
+The receipts would need to be created in the [k/enhancements][k-enhancements] repository. Once a KEP Author/Assignee creates a receipt on behalf of a SIG, it is understood that the feature has been soft opted-in to the release. The opt-in would be finalized only after the Enhancements Team verifies the laid-out acceptance criteria.
+
+The Enhancements Team will continuously monitor the filed receipts during the Enhancements Collection period to verify the graduation requirements. The Enhancements Team will have the power to remove enhancements from the release if they feel they are not conforming to requirements. The team can ask the KEP Assignee to file an exception if the Enhancements Freeze deadline has been passed.
```

Comment:
This feels like a regression from the current process where the sig submits the keps to the enhancements team for inclusion. Feels like the SIG should view/vet the filed receipts _before_ the enhancements team starts looking. Otherwise we're back to authors -> team instead of author->sigs->team. unless receipts require a SIG ACK/NACK before team needs to do anything?

Reply:
Agree on w/ that aspect. I think this should be SIG Chair/TL (or delegate) creates the receipt.